docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -451,6 +470,22 @@ The following table lists the measurement data that the HRM sensor provides.
 | Timestamp                   | `unsigned long long` | -                                   | Microseconds |
 | values[0]: Beats per minute | `int`                | Min. value = 0<br> Max. value = 240 | -            |
 
+<a name="hrm_batch"></a>
+## Heart Rate Monitor Batch Sensor
+
+The Heart Rate Monitor (HRM) Batch sensor measures a person's heart rate, it must be power-efficient and work in a batch manner.
+
+The following table lists the measurement data that the HRM sensor provides.
+
+**Table: Measurement data detected by the HRM sensor**
+
+| Measurement                          | Type                       | Range                               | Unit         |
+|--------------------------------------|----------------------------|-------------------------------------|--------------|
+| Timestamp                            | `unsigned long long`       | -                                   | Microseconds |
+| values[0]: state                     | `sensor_hrm_batch_state_e` | -                                   | -            |
```

Comment:
One doubt : Is s of state intentionally kept in small or else it needs to be in capital letters ?

Reply:
@nawab-samsung I fixed it